src/targets/gpu/fuse_ops.cpp

```diff
@@ -767,11 +767,15 @@ struct find_contiguous
     }
 };
 
-struct find_pointwise_contiguous
+struct find_pointwise_layout
```

Comment:
Maybe this should be named `find_pointwise_layout_contiguous` to make it clear it works for contiguous as well as layout.

Reply:
Done.